node/src/utils.ts

```diff
@@ -26,6 +26,14 @@ export function clone<T>(value: T): T
 	}
 }
 
+/**
+ * Generates a random UUID v4.
+ */
+export function generateUUIDv4(): string
+{
+	return randomUUID();
+}
```

Comment:
Why is that? Why not using directly the `node:crypto` export?

Reply:
To make it clear that it's a UUID version 4 and to be consistent with other utils such as `generateRandomNumber()` which also uses `crypto` API inside. It's not bad to have wrappers that allow us to change the internal implementation at any time if needed.

Reply:
Not my taste of cake, but ok 👍🏻